### ISSUE TITLE
Feature request: show ringtone in list

### ISSUE THREAD

**Issue by johnsgp** [amrmsaraya/clock#4]
In the list of alarms, the ringtone is not shown, so it becomes necessary to edit each one to check it. It would be helpful with my 20+ alarms to see all the ringtones at a glance in the list.

**Comment by amrmsaraya** [comment]
Hello @johnsgp, really appreciate the effort of opening issues. Don't you think if we added the ringtone and snooze duration for every alarm in the list, it will be so much information for our eyes to process?

**Comment by johnsgp** [comment]
Yes, I think you're right there. I'll have to think of a better way to get to the information other than grabbing a pen and paper, but my current suggestion is probably best closed, unless you'd like to keep it open as a discussion point.

The use case is to check at a glance that I've set the correct ringtone for each alarm. For example a morning wake up alarm is loud, but a midday alarm that might ring during a meeting is gentler so as not to disturb the meeting but enough that I'll be notified.  Or an alarm related to a particular person (like picking up a child from school) could be a distinctive one so that it's not necessary to look at the phone to know what it's about.  And so on.  Maybe this could be coordinated with the colour choices or some sort of "alarm profile" instead, which would be a different issue.

**Comment by amrmsaraya** [comment]
Yes, i can't agree more with those use cases.

We can keep this issue open for discussion until we find a better solution.
